packages/contracts/src/resources/site.ts

```diff
@@ -172,6 +189,18 @@ type MaybePubOptions = {
 	 * @default false
 	 */
 	withMembers?: boolean;
+	/**
+	 * Whether to include the legacy assignee.
+	 *
+	 * @default false
+	 */
+	withLegacyAssignee?: boolean;
+	/**
+	 * Whether to include the values.
+	 *
+	 * @default boolean
+	 */
+	withValues?: boolean;
```

Comment:
this speeds up things a lot in places where we only fetch a list of pubs, as we can now rely on the `pub.title` being there